scripts/openshiftci-presubmit-e2e.sh

```diff
@@ -9,7 +9,9 @@ export CI="openshift"
 export TIMEOUT="30m"
 make configure-installer-tests-cluster
 make bin
-export PATH="$PATH:$(pwd)"
+mkdir -p $GOPATH/bin
+go get -u github.com/onsi/ginkgo/ginkgo
+export PATH="$PATH:$(pwd):$GOPATH/bin"
```

Comment:
why adding `$(pwd)`?

Reply:
It was in beginning when the script was created, so i could not analyze the intention of keeping it.

Anyway i do not see any valid reason to keep it. Will remove it

Reply:
Ah, sorry. I haven't noticed that it was there before.

Reply:
$(pwd) is needed in the PATH as we are not copying odo to /usr/bin